README.md

```diff
@@ -199,6 +215,12 @@ This design has the advantage that the tombstone record once written need not be
 the drawback is that in case of a power loss HaloDB cannot guarantee total ordering when put and delete operations are 
 interleaved (although partial ordering for both is guaranteed).
 
+### DB open time
+Open db could take a few minutes, depends on number of records and tombstones. If the db open time is critical to your
+user case, please keep tombstone file size relatively small and increase the number of threads used in building index.
```

Comment:
small comments, can be ignored: `use case` or `user case`? :)

Reply:
```use case``` is better. Will fix it

Reply:
Fixed. Please review https://github.com/wangtao724/HaloDB/tree/build_index